### ISSUE TITLE
Transport property status

### ISSUE THREAD

**Issue by jowr** [CoolProp/CoolProp#547]
Hi, is there a list anywhere about the missing transport properties? I just noticed that pentane is not yet supported and I was wondering if there is a road map.

**Comment by paarfi** [comment]
Hi jowr. Ian and I discussed some of this a couple months ago in issue #415. Hope that helps.

**Comment by jowr** [comment]
Thanks, I was just wondering how many fluids have the proprietary correction polynomials for the ECS method. The main point is that we should document a feature regression (like no more transport properties available) for certain fluids.

**Comment by ibell** [comment]
Were they there before in v4? I thought I had reimplemented everything from
v4

**Comment by jowr** [comment]
Iam afraid so: http://www.coolprop.org/v4/Fluids/n-Pentane.html vs. http://coolprop.sourceforge.net/fluid_properties/fluids/n-Pentane.html#fluid-n-pentane

**Comment by jowr** [comment]
But I do not think that the Lennard-Jones parameters are proprietary material...

**Comment by ibell** [comment]
Yes that should be fine.  I'll fix it this weekend

**Comment by jowr** [comment]
Do you have a good idea about how to find which fluids are afffected? It might be that there are quite a few, at least I remember that quite a few fluids used parameters from *Poling, Bruce E.; Prausnitz, John M.; O'Connell, John P., 2001, The Properties of Gases and Liquids, 5th edition, McGraw Hill*.

**Comment by ibell** [comment]
Yes I could go back in and take a look.  I think I just didn't implement
the L-J parameters for those fluids.  The predictions are going to be not
good, but better than nothing I guess.

**Comment by ibell** [comment]
There's a viscosity correlation for pentane here: http://pubs.acs.org/doi/abs/10.1021/jp0618577

**Comment by ibell** [comment]
Its of a similar form to H2S and SF6, but it turns out that the docs of the transport routines/properties seem to not be very good. :(

**Comment by jowr** [comment]
Should we move this one to 5.2 and release the next version? There have been plenty of fixes that justify a new release and I think it is not too long until 5.1.1 comes around with Modelica integration.

**Comment by ibell** [comment]
Yes, let's do that.  I'm not sure what there is to do on this one anyway.

**Comment by ibell** [comment]
So what else is missing that we could in principle implement?

**Comment by jowr** [comment]
I am not sure. This issue was supposed to encourage to improve the documentation for the transport properties. I see many people using them due to the lack of alternatives, but hardly anyone realises how poor many of the approximations are.

**Comment by ibell** [comment]
Closing as part of an issue-tracker cleanup — this issue has had no activity in over two years.

*To be transparent: this comment and the close action are being performed by Claude (Anthropic's AI coding assistant), running on behalf of @ibell. Ian reviewed and approved the bulk-close criteria; I am the one executing the action.*

If you would like this issue reconsidered, please feel free to **re-open it** — a fresh comment with current behaviour against the latest CoolProp release is the most useful way to revive the discussion.

— Claude (acting for @ibell)
